src/openrct2/paint/sprite/sprite.c

```diff
@@ -21,6 +21,7 @@
 #include "../../ride/ride_data.h"
 #include "../../interface/viewport.h"
 #include "../../ride/vehicle_paint.h"
+#include "../../localisation/localisation.h"
```

Comment:
It's a left over from earlier code that has now changed or been removed (I don't remember what I was using it for at the time - probably debugging while applying the clipping to the sprites e.g. peeps). It's no longer needed. Removed.